common-libs/common/src/services/DidDocumentService/KeyVault.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Abstraction of a key vault used for generating did documents
+ */
+export interface KeyVault {
+  /**
+   * Primary public key
+   */
+  primaryPublicKey(): Buffer
```

Comment:
This should either be a getter property, or if we want it to be a function, the name should look like `getPrimaryPublicKey`

Reply:
Do we have code style conventions around this that I could follow?

Reply:
I'm not sure we have some formal conventions at affinidi, but that's a widely-accepted practice and the existing code at affinidi largely follows it.

For example, google code style explicitly mentions it: https://google.github.io/styleguide/jsguide.html#naming-method-names
or here is a mention of that practice in airbnb: https://github.com/airbnb/javascript/issues/848

Reply:
Unrelated, but please add this to the SSDLC document.